RecoLocalMuon/CSCRecHitD/src/CSCHitFromWireOnly.cc

```diff
@@ -76,7 +80,17 @@ std::vector<CSCWireHit> CSCHitFromWireOnly::runWire( const CSCDetId& id, const C
       std::vector <int> timeBinsOn=wire_cluster[n_wgroup/2].getTimeBinsOn();
       //CSCWireHit whit(id, whit_pos, wire_in_clusterAndBX, theTime, isDeadWGAround, timeBinsOn );
       CSCWireHit whit(id, whit_pos, wire_in_clusterAndBX, theTime, aDeadWG, timeBinsOn );
-      hitsInLayer.push_back( whit );	
+
+      if (!useReducedWireTime) {
+
+         hitsInLayer.push_back( whit );
+
+         } else if (theTime >= wireTimeWindow_low && theTime <= wireTimeWindow_high) {
+
+                   hitsInLayer.push_back( whit );	
+
+                   }
```

Comment:
please use consistent indentation.
also, most of the empty lines are not needed in the added code